libafl_sugar/src/forkserver.rs

```diff
@@ -141,6 +144,66 @@ impl ForkserverBytesCoverageSugar<'_> {
                     .track_indices()
             };
 
+            // New maximization map feedback linked to the edges observer and the feedback state
+            let map_feedback = MaxMapFeedback::with_name("map_feedback", &edges_observer);
+            // Extra MapFeedback to deduplicate finds according to the cov map
+            let map_objective = MaxMapFeedback::with_name("map_objective", &edges_observer);
+
+            let calibration = CalibrationStage::new(&map_feedback);
+
+            let stats_stage = AflStatsStage::builder()
+                .map_observer(&edges_observer)
+                .build()?;
+
+            let colorization = ColorizationStage::new(&edges_observer);
+
+            // The cmplog map shared between observer and executor
+            let mut cmplog_shmem = shmem_provider_client
+                .uninit_on_shmem::<AFLppCmpLogMap>()
+                .unwrap();
+            // let the forkserver know the shmid
+            unsafe {
+                cmplog_shmem.write_to_env(SHM_CMPLOG_ENV_VAR).unwrap();
+            }
+            let cmpmap = unsafe { OwnedRefMut::from_shmem(&mut cmplog_shmem) };
+
+            let cmplog_observer = AFLppCmpLogObserver::new("cmplog", cmpmap, true);
+            let cmplog_ref = cmplog_observer.handle();
+
+            let cmplog_executor = ForkserverExecutor::builder()
+                .program(self.program.clone())
+                .parse_afl_cmdline(self.arguments)
+                .is_persistent(true)
+                .coverage_map_size(MAP_SIZE)
+                // increase timeouts for cmplog
+                .timeout(timeout * 10)
+                .debug_child(self.debug_output)
+                .shmem_provider(&mut shmem_provider_client)
+                .build(tuple_list!(cmplog_observer))
+                .unwrap();
+
+            let tracing = AFLppCmplogTracingStage::new(cmplog_executor, cmplog_ref);
```

Comment:
don't use aflpp cmplog it's not good

Reply:
This is just copy&paste from forkserver_fuzzbench, shouldn't this be good enough? It's still better than no cmplog

Reply:
no there's the normal cmplog that do stuff in a non-deterministic manner

Reply:
How do I use that in forkserver

Reply:
https://github.com/AFLplusplus/LibAFL/blob/main/fuzzers/forkserver/fuzzbench_forkserver/src/main.rs
this one